src/Uno.UI/Extensions/UIViewExtensions.iOSmacOS.cs

```diff
@@ -295,7 +287,18 @@ public static void AddChild(this _View parent, _View child)
 		/// <summary>
 		/// Get the parent view in the visual tree. This may differ from the logical <see cref="FrameworkElement.Parent"/>.
 		/// </summary>
-		public static _View GetVisualTreeParent(this _View child) => child?.Superview;
+		public static _View GetVisualTreeParent(this _View child)
+		{
+			var visualParent = child?.Superview;
+
+			if (visualParent is null &&
```

Comment:
I would add a comment to explicit the cases where the `visualParent` could be null while `fw.VisualPrent` is not.